core/src/components/picker-column-internal/picker-column-internal.tsx

```diff
@@ -251,6 +254,7 @@ export class PickerColumnInternal implements ComponentInterface {
 
           if (selectedItem.value !== this.value) {
             this.value = selectedItem.value;
+            this.emitIonChange();
```

Comment:
We should consider having a `setValue` method that is used internally that encapsulated setting `this.value` and emitting the change event. It sounds like any time we change `value` internally we also need to emit `ionChange`?